config_defaults/subtests/docker_cli/run_volumes.ini

```diff
@@ -2,6 +2,9 @@
 subsubtests = volumes_rw, volumes_one_source
 #: These must be setup prior to testing
 __example__ = host_paths, cntr_paths
+#: Command(s) required to succeede prior to testing (if any)
+#: e.g. setsebool virt_sandbox_use_nfs
```

Comment:
I think it can be separated however you like, it's just passed through to ``utils.run()`` which IIRC sends it into a 'bash -c' as a string.  Unless I'm misunderstanding your comment.